tests/utilities/test_reader.py

```diff
@@ -528,7 +528,7 @@ def test_pvdreader_no_part_group():
 
     reader.set_active_time_value(1.0)
     assert len(reader.active_datasets) == 2
-    for i, dataset in enumerate(reader.active_datasets):
+    for _i, dataset in enumerate(reader.active_datasets):
```

Comment:
I think using `ii` would be better here (and elsewhere).

Reply:
I guess we can remove `i` here.
```suggestion
    for dataset in reader.active_datasets:
```